torchao/prototype/quantized_training/README.md

```diff
@@ -46,8 +46,46 @@ BF16 compile    | 10.16915
 INT8 QT eager   | 10.11437
 INT8 QT compile | 10.03365
 
+## INT8 mixed-precision
+
+On NVIDIA GPUs, INT8 Tensor Cores can be up to 3x faster than their BF16/FP16 counterparts. In mixed-precision training, we can down-cast activations and weights dynamically to INT8 to leverage faster matmuls. However, since INT8 has very limited range [-128,127], we perform row-wise quantization, similar to how INT8 post-training quantization (PTQ) is done. Weight is still in original precision. This is inspired by prior works:
```

Comment:
wondering if you have any context on why we see a 3x peak speedup and not a 2x peak speedup?

Reply:
I did microbenchmark here https://github.com/gau-nernst/quantized-training?tab=readme-ov-file#matmul on my GPU (sm89). Using both CuBLAS (via `torch._int_mm()`) and triton matmul. Perhaps I can add that this speed up is measured on my specific GPU (4070Ti SUPER, sm89). I can probably spin up an A100 instance and do the microbenchmark there too.

Reply:
I can also add an INT8 microbenchmark script here, similar to `benchmarks/float8/bench_matmul.py`.

Reply:
Added the INT8 microbenchmark and its results in the PR description. Surprisingly, I only observe 3x speedup in consumer sm89 cards. Even L40s (also sm89) doesn't see such speed up :thinking:

I updated the README to "2x faster" so it's more conservative.